modules/equativBidAdapter.js

```diff
@@ -0,0 +1,153 @@
+import { BANNER } from '../src/mediaTypes.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { deepAccess, deepSetValue, isFn, mergeDeep } from '../src/utils.js';
+
+/**
+ * @typedef {import('../src/adapters/bidderFactory.js').Bid} Bid
+ * @typedef {import('../src/adapters/bidderFactory.js').ServerRequest} ServerRequest
+ */
+
+export const spec = {
+  code: 'equativ',
+  gvlid: 45,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * @param bidRequests
+   * @param bidderRequest
+   * @returns {ServerRequest[]}
+   */
+  buildRequests: (bidRequests, bidderRequest) => {
+    return {
+      data: converter.toORTB({ bidderRequest, bidRequests }),
+      method: 'POST',
+      url: 'https://ssb-global.smartadserver.com/api/bid?callerId=169'
+    };
+  },
+
+  /**
+   * @param bidRequest
+   * @returns {number}
+   */
+  getMinFloor: (bidRequest) => {
```

Comment:
You guys have this same code block in two adapters, please import it

Reply:
Done.